engine/apps/api/serializers/on_call_shifts.py

```diff
@@ -36,7 +36,7 @@ class Meta:
         fields = [
             "id",
             "organization",
-            "name",
+            "title",
```

Comment:
I guess we will need to include the `updated_shift` field too, read-only, to indicate when there is an updated version of the current shift (eventually, frontend should decide to show a warning or "redirect" the user to edit the latest version of the shift)

Reply:
makes sense